tests/cloner_test.go

```diff
@@ -1029,6 +1029,90 @@ var _ = Describe("all clone tests", func() {
 				Expect(utils.GetCloneType(f.CdiClient, dataVolume)).To(Equal("csivolumeclone"))
 			})
 		})
+
+		Context("Clone without a source PVC", func() {
+			diskImagePath := filepath.Join(testBaseDir, testFile)
+			fsVM := v1.PersistentVolumeMode(v1.PersistentVolumeFilesystem)
+			blockVM := v1.PersistentVolumeMode(v1.PersistentVolumeBlock)
+
+			deleteAndWaitForVerifierPod := func() {
+				By("Deleting verifier pod")
+				err := utils.DeleteVerifierPod(f.K8sClient, f.Namespace.Name)
+				Expect(err).ToNot(HaveOccurred())
+				_, err = utils.WaitPodDeleted(f.K8sClient, utils.VerifierPodName, f.Namespace.Name, verifyPodDeletedTimeout)
+				Expect(err).ToNot(HaveOccurred())
+			}
+
+			compareCloneWithSource := func(sourcePvc, targetPvc *v1.PersistentVolumeClaim, sourceImgPath, targetImgPath string) {
```

Comment:
Can you split this into two functions, my main motivation is that currently it is possible that if `f.GetMD5` errors, the verifier pod is never cleaned up by the test. This is not a huge issue because we delete the namespace at the end of the test, but I want to be diligent about cleaning up things we create without having to rely on the namespace to do it. If you split this, you can `defer` the `deleteAndWaitForVerifierPod` and be sure it is called.